front/pages/api/w/[wId]/apps/[aId]/runs/index.ts

```diff
@@ -79,89 +79,6 @@ async function handler(
       ]);
 
       switch (req.body.mode) {
```

Comment:
we can remove the body.mode entirely no?

Reply:
Done